pyproject.toml

```diff
@@ -30,6 +31,10 @@ ipykernel = "^5.3.4"
 ipython = "^7.16.1"
 pdoc3 = "^0.8.4"
 pandas = "^1.1.4"
+dask = {extras = ["distributed"], version = "^2021.2.0"}
+bokeh = "^2.2.3"
+jupyter-server-proxy = "^1.6.0"
+jupyterlab = "^3.0.7"
```

Comment:
idem

Reply:
I'll remove `jupyter-server-proxy` (don't remember why I added it at a certain point), but I rather keep `jupyterlab` as it is my current platform for `.ipynb` notebooks. Not sure if there is a better way to handle this situation - namely a package that is a `dev dependency`, but just for me as a "user", not for the package itself.

Reply:
Best practice is to have a global jupyterlab install and add the poetry .venv to your kernels.

Reply:
check.